### PR TITLE
Change domain form display options

### PR DESCRIPTION
**Pull request by labkey-ankurj**
#### Rationale
This PR is to pull ui-components version of domain form display options fix.

#### Related Pull Requests
https://github.com/LabKey/labkey-ui-components/pull/240

### CODE REVIEW

**Review by cnathe — APPROVED** [review]

File: query/src/client/QueryMetadataEditor/QueryMetadataEditor.tsx [hunk 1]
```diff
@@ -328,11 +328,11 @@ export class App extends PureComponent<any, Partial<IAppState>> {
...
                         useTheme={false}
                         domainFormDisplayOptions= {{
-                            showRequired: false,
+                            hideRequired: true,
                             isDragDisabled: true,
-                            showValidators: false,
+                            hideValidators: true,
                             phiLevelDisabled: true,
-                            showAddFieldsButton: false
+                            hideAddFieldsButton: true,
```
Comment (cnathe) [inline query/src/client/QueryMetadataEditor/QueryMetadataEditor.tsx:335]: Shouldn't this also set `hideTextOptions: true`?

**Review by cnathe — CHANGES_REQUESTED** [review]

File: query/src/client/QueryMetadataEditor/QueryMetadataEditor.tsx (hunk 1 above)
Comment (labkey-ankurj) [reply query/src/client/QueryMetadataEditor/QueryMetadataEditor.tsx:335]: it wasn't set to showTextOptions: false before, I don't recall the text options in old gwt editor. I can change if you think we need to hide them.

File: core/package-lock.json [hunk 2]
```diff
@@ -11005,7 +11005,7 @@
     },
     "rgba-regex": {
       "version": "1.0.0",
-      "resolved": "https://registry.npmjs.org/rgba-regex/-/rgba-regex-1.0.0.tgz",
+      "resolved": "http://registry.npmjs.org/rgba-regex/-/rgba-regex-1.0.0.tgz",
```
Comment (cnathe) [inline core/package-lock.json]: @labkey-ankurj  Need to revert this change. It usually can be fixed by clearing the node_modules for core and then rebuilding that module.

Comment (cnathe) [reply core/package-lock.json]: done.

**Review by cnathe — APPROVED** [review]

**Pull request merged**